From fb1a884a811c58791211fa0e51d29c09eb802baf Mon Sep 17 00:00:00 2001
From: Leonora Tindall <nora@nora.codes>
Date: Mon, 8 Jul 2019 11:53:20 -0700
Subject: [PATCH] Fix wrong/not uninstalled icons in Makefile

---
 Makefile | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index afd033d..f02937b 100644
--- a/Makefile
+++ b/Makefile
@@ -33,8 +33,9 @@ install : target/release/gDiceRoller
 	mkdir -p $(sharedir)/metainfo/
 	# Install icons
 	$(INSTALL_DATA) data/codes.nora.gDiceRoller.svg $(sharedir)/icons/hicolor/scalable/apps/codes.nora.gDiceRoller.svg
-	$(INSTALL_DATA) data/codes.nora.gDiceRoller.64.png $(sharedir)/icons/hicolor/64x64/apps/codes.nora.gDiceRoller.svg
-	$(INSTALL_DATA) data/codes.nora.gDiceRoller.128.png $(sharedir)/icons/hicolor/128x128/apps/codes.nora.gDiceRoller.svg
+	$(INSTALL_DATA) data/codes.nora.gDiceRoller.64.png $(sharedir)/icons/hicolor/64x64/apps/codes.nora.gDiceRoller.png
+	$(INSTALL_DATA) data/codes.nora.gDiceRoller.128.png $(sharedir)/icons/hicolor/128x128/apps/codes.nora.gDiceRoller.png
+	# Force icon cache refresh
 	touch $(sharedir)/icons/hicolor
 	# Install application meta-data
 	$(INSTALL_DATA) data/codes.nora.gDiceRoller.appdata.xml $(sharedir)/metainfo/codes.nora.gDiceRoller.appdata.xml
@@ -49,6 +50,8 @@ uninstall :
 	rm -f $(sharedir)/metainfo/codes.nora.gDiceRoller.appdata.xml
 	# Remove the icon
 	rm -f $(sharedir)/icons/hicolor/scalable/apps/codes.nora.gDiceRoller.svg
+	rm -f $(sharedir)/icons/hicolor/64x64/apps/codes.nora.gDiceRoller.png
+	rm -f $(sharedir)/icons/hicolor/128x128/apps/codes.nora.gDiceRoller.png
 	# Remove the binary
 	rm -f $(bindir)/bin/codes.nora.gDiceRoller
 
-- 
GitLab

